server/integration/infinispan/src/main/resources/org/jboss/as/clustering/infinispan/subsystem/LocalDescriptions.properties

```diff
@@ -439,8 +438,7 @@ datagrid-infinispan.store.passivation=If true, data is only written to the cache
 datagrid-infinispan.store.passivation.deprecated=Deprecated please use the passivation attribute on the persistence model.
 datagrid-infinispan.store.fetch-state=If true, fetch persistent state when joining a cluster. If multiple cache stores are chained, only one of them can have this property enabled.
 datagrid-infinispan.store.purge=If true, purges this cache store when it starts up.
-datagrid-infinispan.store.singleton=If true, the singleton store cache store is enabled. SingletonStore is a delegating cache store used for situations when only one instance in a cluster should interact with the underlying store.
-datagrid-infinispan.store.singleton.deprecated=Deprecated. A shared store should be used instead, as this limits store writes to the primary owner of a key.
+datagrid-infinispan.store.singleton.deprecated=Ignored. A shared store should be used instead, as this limits store writes to the primary owner of a key.
```

Comment:
This can't be removed as well?

Reply:
Ah guess maybe not, the others have it as well. That seems like a lot of junk to carry around.

Reply:
You were right here @wburns, and after checking again I found a few more things I missed: https://github.com/infinispan/infinispan/pull/7014